doc/tutorials/calib3d/camera_multiview_calibration/multiview_calibration.markdown

```diff
@@ -0,0 +1,226 @@
+Multi-view Camera Calibration Tutorial
+==================
+
+The following sections describes the individual algorithmic steps of the overall multi-camera calibration pipeline:
```

Comment:
I propose to start from the problem statement. We have several cameras that have FoW intersection and want to estimate ther internal parameters (intrinsics) and relative positions (extrinsics) ....

Reply:
Also need to point that the cameras should be in sync otherwise the solution does not work.

Reply:
fixed